File: Client/Pikachu/PikachuLib/Communication/Models/Usermanagement/CreateUserResponse.cs
using Newtonsoft.Json;

namespace PikachuLib.Communication.Models.Usermanagement
{
    /// <summary>
    /// Repräsentation eines Create User Response
    /// </summary>
    public class CreateUserResponse
    {
        [JsonProperty("id")]
        public int Id { get; set; }
    }
}

File: Server/relaxo/src/main/java/de/teamrocket/relaxo/rest/models/usermanagement/response/CreateUserResponse.java
package de.teamrocket.relaxo.rest.models.usermanagement.response;

/**
 * Created by henry on 17.01.15.
 */
public class CreateUserResponse {

    private int id;

    public CreateUserResponse(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }
}
